### DISCUSSION TITLE
Game-Changer for GPS Lock: My Experience with iNav 8 RC3

### DISCUSSION THREAD

**Discussion by ivolanski** [iNavFlight/inav#10549] [General]
Hi All,

I wanted to share an exciting breakthrough I recently experienced while using iNav on my RC car. For a long time, I struggled with GPS lock issues. My setup consistently had difficulty maintaining a reliable satellite connection, which made GPS-based features almost unusable.

Here’s what I was dealing with:

The GPS would take forever to lock onto even 5 satellites.
It constantly bounced between 3 and 7 satellites, making the data unreliable.
I tried everything: moving the GPS module to a clear sky view, shortening the wires, and distancing it as far as possible from the video transmitter. Despite all these efforts, the problem persisted.
Today, I decided to give iNav 8 RC3 a try, and I enabled AssistNow offline. The difference was nothing short of incredible—like going from water to wine!

I got an instant and stable lock with 20+ satellites.
The connection remained consistent throughout the session.
This improvement has been a complete game-changer for my RC car experience. I want to take a moment to praise and thank the iNav development team for this fantastic enhancement. Your hard work has genuinely improved the quality of my RC life, and I’m incredibly grateful to everyone involved in making this happen.

If anyone out there is struggling with GPS lock issues, I highly recommend trying iNav 8 RC3 and enabling AssistNow offline. It might just transform your experience too!

Thanks again to the entire iNav team—you’ve truly outdone yourselves!

**Comment by mmosca** [comment]
Thank you. This made my day.
